src/WebJobs.Script/WebJobs.Script.csproj

```diff
@@ -41,11 +41,12 @@
     <DocumentationFile>
     </DocumentationFile>
   </PropertyGroup>
+  <PropertyGroup Condition="'$(OS)'!='Unix'">
+    <EnableNode>True</EnableNode>
+    <EnablePowerShell>True</EnablePowerShell>
+    <DefineConstants>$(DefineConstants);WINDOWS;FEATURE_NODE;FEATURE_POWERSHELL</DefineConstants>
```

Comment:
I'm hoping we can simplify all these various switches (MONO,WINDOWS,FEATURE_NODE,FEATURE_POWERSHELL) to just a single MONO switch. We're not going to actually having all these various permutations right? We can just simplify all these conditionals in the code to use just a single MONO constant.